build/lib/setup.sh

```diff
@@ -19,7 +19,7 @@ if [ ! -d "/root/.docker" ]; then
     mkdir -p /root/.docker
 fi
 
-mv config/docker-ecr-config.json /root/.docker/config.json
+cp config/docker-ecr-config.json /root/.docker/config.json
```

Comment:
the mv was causing this to show as a git diff which complicated the rebase, the cp shouldnt make a difference.